docs/podman-container.1.md

```diff
@@ -14,6 +14,7 @@ The container command allows you to manage containers
 | Command  | Man Page                                            | Description                                                                  |
 | -------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
 | attach   | [podman-attach(1)](podman-attach.1.md)              | Attach to a running container.                                               |
+| cleanup  | [podman-container-cleanup(1)](podman-container-cleanup.1.md)    | Cleanup containers network and mountpoints.                               |
```

Comment:
This man-page URL includes `-container`, which is missing from the surrounding entries.  Do those files need to be renamed to include it?

Reply:
No, podman container attach == podman attach, it is just a different way of executing the same code.

BTW
docker container attach == docker attach also

Reply:
> No, podman container attach == podman attach...

And similarly for all of the other commands here?  It looks like `refresh` (at `podman-container-refresh.1.md`) is the only existing `container` subcommand that has a `container`-specific man-page.

Reply:
Yes we could create some links for the others, so that `man podman container attach` would show you the `podman attach` man page.